File: backendworker/NHSOnline.Backend.GpSystems/Suppliers/Tpp/Models/Appointments/BookAppointmentReply.cs
using System;
using System.Xml.Serialization;

namespace NHSOnline.Backend.GpSystems.Suppliers.Tpp.Models.Appointments
{
    [Serializable]
    public class BookAppointmentReply
    {
        [XmlAttribute("patientId")]
        public string PatientId { get; set; }

        [XmlAttribute("onlineUserId")]
        public string OnlineUserId { get; set; }

        [XmlAttribute("uuid")]
        public string UuId { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/BookAppointmentReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement

@XmlAccessorType(XmlAccessType.FIELD)
@XmlRootElement(name = "BookAppointmentReply")
data class BookAppointmentReply(
        @XmlAttribute var patientId: String = "default patientId",
        @XmlAttribute var onlineUserId: String = "default onlineUserId",
        @XmlAttribute var message: String = "default message",
        @XmlAttribute var uuid: String = "default uuid"
)
